action.yaml

```diff
@@ -10,10 +10,14 @@ inputs:
   token:
     description: 'Your GitHub Registry token'
     required: true
+  repo:
```

Comment:
should it be something like `registry_namespace` instead?

Reply:
yes, makes more sense